sorc/fv3nc2nemsio.fd/makefile.sh

```diff
@@ -3,6 +3,21 @@ set -x
 
 machine=${1:-"cray"}
 
+if [ $machine = "cray" ]; then
+:
+elif [ $machine = "theia" ]; then
```

Comment:
Should this be "hera" instead of "theia"?

Reply:
This came from when I was working with gfsv15 (port2hera) branch that did not know about hera, and has these safeguards.  This part of the code is removed in gfsv16b, so I need to remove the if conditions all in all. For the future though, we need to think about how auto-merging can lead to these kinds of errors. I will fix this shortly, thanks!